todolist/src/components/TodoContainer.js

```diff
@@ -3,84 +3,164 @@ import TodosList from "./TodosList";
 import Header from "./Header"
 import InputTodo from "./InputTodo"
 import { v4 as uuidv4 } from "uuid";
+// class TodoContainer extends React.Component {
+//     state = {
+//         todos: [
+//           {
+//             id: uuidv4(),
+//             title: "Setup the development environment",
+//             completed: true
+//           },
+//           {
+//             id: uuidv4(),
+//             title: "Develop the website and add content",
+//             completed: false
+//           },
+//           {
+//             id: uuidv4(),
+//             title: "Deploy it to live server",
+//             completed: false
+//           }
+//         ]
+//        };
+//        handleChange = id => {
+//         // this.setState(prevState => ({
+//         //     todos: prevState.todos.map(todo => {
+//         //       if (todo.id === id) {
+//         //         todo.completed = !todo.completed
+//         //       }
+//         //       return todo
+//         //     }),
+//         //   }))
+//         // . The code above wont work because we're  using "strictMode" on the index.js file
+//         this.setState(prevState => ({
+//             todos: prevState.todos.map(todo => {
+//               if (todo.id === id) {
+//                 return {
+//                   ...todo,
+//                   completed: !todo.completed,
+//                 }
+//               }
+//               return todo
+//             }),
+//           }))
+//       };
+
+//       delTodo = id => {
+//         this.setState({
+//           todos: [
+//             ...this.state.todos.filter(todo => {
+//               return todo.id !== id;
+//             })
+//           ]
+//         });
+//       };
+
+//       addTodoItem = title => {
+//         const newTodo = {
+//             id: uuidv4(),
+//           title: title,
+//           completed: false
+//         };
+//         this.setState({
+//           todos: [...this.state.todos, newTodo]
+//         });
+//       };
+
+//       render() {
+//         return (
+//           <div className="container">
+//             <div className="inner">
+//               <Header />
+//               <InputTodo addTodoProps={this.addTodoItem} />
+//               <TodosList
+//                 todos={this.state.todos}
+//                 handleChangeProps={this.handleChange}
+//                 deleteTodoProps={this.delTodo}
+//               />
+//             </div>
```

Comment:
- [ ] [OPTIONAL] There is almost an entire file of commented code. Please remove them if they are not needed, as this can make your code unorganised, and difficult to debug as linters cannot read such codes.